cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We need to figure out what to put here when we start using it

Reply:
updated to add baseURL of localhost:19006